server/app/services/PhoneValidationUtils.java

```diff
@@ -0,0 +1,45 @@
+package services;
+
+import com.google.i18n.phonenumbers.NumberParseException;
+import com.google.i18n.phonenumbers.PhoneNumberUtil;
+import com.google.i18n.phonenumbers.Phonenumber;
+import java.util.Optional;
+
+/** Utility Class for all phone validations. */
+public final class PhoneValidationUtils {
+
+  private static final PhoneNumberUtil PHONE_NUMBER_UTIL = PhoneNumberUtil.getInstance();
+
+  public static Optional<MessageKey> validatePhoneNumber(
```

Comment:
Suggestion (blocking): Can you add some documentation specifying what this method returns and when? E.g. `@returns a MessageKey representing the validation error that occurred if the phone number is invalid, or an Optional.empty() if the phone number is valid`.